pkg/kclient/deployments.go

```diff
@@ -137,3 +139,20 @@ func (c *Client) UpdateDeployment(deploymentSpec appsv1.DeploymentSpec) (*appsv1
 	}
 	return deploy, nil
 }
+
+// CreateDynamicDeployment creates a dynamic deployment for Operator backed service
+func (c *Client) CreateDynamicDeployment(exampleCR map[string]interface{}, group, version, resource string) error {
```

Comment:
This method can be used to create any resource not just a `exampleCR`, can we rename the func attribute